llvm/lib/Transforms/Scalar/ConstraintElimination.cpp

```diff
@@ -748,11 +749,22 @@ void ConstraintInfo::transferToOtherSystem(
   default:
     break;
   case CmpInst::ICMP_ULT:
-    //  If B is a signed positive constant, A >=s 0 and A <s B.
+  case CmpInst::ICMP_ULE:
+    //  If B is a signed positive constant, A >=s 0 and A <s (or <=s) B.
```

Comment:
I think adding a then would make the sentence easier to parse.


```suggestion
    //  If B is a signed positive constant, then A >=s 0 and A <s (or <=s) B.
```